src/components/Shared/SendModal/SendLN/SendLN.tsx

```diff
@@ -7,24 +7,35 @@ const SendLn: FC<SendLnProps> = (props) => {
   const { balance, onChangeInvoice, onConfirm } = props;
   const { t } = useTranslation();
 
+  const [isFormValid, setIsFormValid] = useState(true);
+
+  const validateInput = (event: ChangeEvent<HTMLInputElement>) => {
+    // TODO: check wallet balance
+    setIsFormValid(event.target.validity.valid)
+  }
+
   return (
     <form onSubmit={onConfirm}>
       <h3 className="text-xl font-bold">{t("wallet.send_lightning")}</h3>
-      <div className="my-5">
+      <p className="my-5">
         <span className="font-bold">{t("wallet.balance")}:&nbsp;</span>
         {balance} {appCtx.unit}
-      </div>
+      </p>
       <label className="label-underline" htmlFor="invoiceInput">
         {t("wallet.invoice")}
       </label>
       <input
         id="invoiceInput"
         type="text"
         onChange={onChangeInvoice}
-        className="input-underline"
+        onBlur={validateInput}
```

Comment:
This is to discuss if `onChange` or `onBlur` should be used.

I *would* be in favor of `onChange` since you get immediate feedback and you don't have to "click out" after entering the invoice.